code/datums/emotes.dm

```diff
@@ -91,19 +91,18 @@
 /datum/emote/proc/select_param(mob/user, params)
 	return replacetext(message_param, "%t", params)
 
-/datum/emote/proc/can_run_emote(mob/user, help_check)
+/datum/emote/proc/can_run_emote(mob/user)
 	. = TRUE
 	if(!is_type_in_typecache(user, mob_type_allowed_typecache))
 		return FALSE
 	if(is_type_in_typecache(user, mob_type_blacklist_typecache))
 		return FALSE
-	if(!help_check)
```

Comment:
you can't just remove this; it exists so that emotes you can potentially use but are currently unable to use still appear when you use `*help`

Reply:
Ah fuck, I didn't notice the !

That means all this was already working and something else is fucked up. Now I've actually got to test this shit fuck.